src/compiler/types.ts

```diff
@@ -1869,6 +1875,7 @@ namespace ts {
 
     export interface JSDocTypedefTag extends JSDocTag, Declaration {
         kind: SyntaxKind.JSDocTypedefTag;
+        fullName?: JSDocNamespaceDeclaration | Identifier;

```

Comment:
Can you reuse or wrap `NamespaceDeclaration` instead of making a new type `JSDocNamespaceDeclaration`? I'd like to  eventually remove all duplication between typescript and jsdoc nodes.